readme.md

```diff
@@ -0,0 +1,30 @@
+# Engine Test Data
```

Comment:
As above, this file should not be contained in this PR - it should be included via a git submodule.

Reply:
I'll fix this.